modules/rubiconAnalyticsAdapter.js

```diff
@@ -73,7 +73,7 @@ function sendMessage(auctionId, bidWonId) {
   function formatBid(bid) {
     return utils.pick(bid, [
       'bidder',
-      'bidId',
+      'bidId', bidId => (utils.deepAccess(bid, 'bidResponse.seatBidId')) ? utils.deepAccess(bid, 'bidResponse.seatBidId') : bidId,
```

Comment:
this works fine, but more concise to use `bidId => utils.deepAccess(bid, 'bidResponse.seatBidId') || bidId,`